tests/neg/derive-eq.scala

```diff
@@ -20,10 +20,10 @@ object Test extends App {
   val y: Triple[List[Two], One, Two] = ???
   val z: Triple[One, List[Two], One] = ???
   x == y // OK
+  y == x // OK
```

Comment:
I think this should work in this way. If `x == y` is valid `y == x` should also be valid.